sparx/templates/wikiparser.py

```diff
@@ -2276,6 +2284,7 @@ def build_config_list_DokuWiki(is_dev_mode = False):
 	sxcmd_config_list.append(SXcmd_config("../doc/e2display.txt", "DokuWiki", sxcmd_category, sxcmd_role, exclude_list = create_exclude_list_display(), is_submittable = False))
 	sxcmd_config_list.append(SXcmd_config("../doc/pipe_organize_micrographs.txt", "DokuWiki", sxcmd_category, sxcmd_role))
 	sxcmd_config_list.append(SXcmd_config("../doc/batch.txt", "DokuWiki", sxcmd_category, sxcmd_role, is_submittable = False))
+	sxcmd_config_list.append(SXcmd_config("../doc/gui_ctfrefine.txt", "DokuWiki", sxcmd_category, sxcmd_role, is_submittable=False))
```

Comment:
This should be submittable, just without MPI (for now).

Add 
sxctf_refine.py exists only in non-MPI version.

to (the renamed) ctf_refine.txt in the Typical usage section (like https://github.com/cryoem/eman2/blob/master/sparx/doc/unblur.txt)

Reply:
Done.